cudax/include/cuda/experimental/__event/event.cuh

```diff
@@ -24,13 +24,12 @@
 #  pragma system_header
 #endif // no system header
 
+#include <cuda/experimental/__detail/utility.cuh>
+#include <cuda/experimental/__event/event_ref.cuh>
 #include <cuda/std/__cuda/api_wrapper.h>
 #include <cuda/std/cstddef>
 #include <cuda/std/utility>
 
-#include <cuda/experimental/__detail/utility.cuh>
-#include <cuda/experimental/__event/event_ref.cuh>
-
 namespace cuda::experimental
```

Comment:
Can we add the clang-format changes so that w have the experimental headers last?

Reply:
Will be addressed here: #2094